packages/x-components/src/components/result/base-result-image.vue

```diff
@@ -31,7 +32,7 @@
         :key="imageSrc"
         :alt="result.name"
         :src="imageSrc"
-        class="x-result-picture-image"
+        class="x-result-picture-image x-picture-image"
```

Comment:
I think that the `x-picture-image` class is not needed here, it will be appended from the dynamic component tag.

Reply:
I'm not catching you, before we had the `aspect-ratio` and other styles coming from the `img` tag styles attribution made on XDS. In the last related [PR](https://github.com/empathyco/x/pull/1035) we changed from `img` to use `class`, so I applied here he class accordingly.

Reply:
I think that I didn't explain myself very well xd
The `x-picture-image` class is already set, the `<component :is="animation"/>` tag will render a Vue Transition or a NoElement.
Because these two are renderless components, `Vue` will inherit the attr class and set it to the element that is rendering.
You can test this by removing the class from the image and inspecting the element in the devtools, it should contain the class because it's been set in the `component` parent.

Reply:
I've seen it now! Thks!!!